client/components/Title.jsx

```diff
@@ -11,4 +11,4 @@ const Title = ({
   </div>
```

Comment:
Please revert changes back to original. (I have similar fixes on my current branch.)